### PR TITLE
feat: use modified minidom to avoid MissingNamespace error

### LINKED ISSUE #2154: Ensure that StringParseXML supports APIs that have no "xmlns" attribute
Check that the StringParseXML operator supports the format returned by this API:

https://forecast.weather.gov/MapClick.php?lat=39.75&lon=-104.99&unit=0&lg=english&FcstType=dwml

### PR DESCRIPTION
**Pull request by lrubiorod**
Close #2154

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -2535,31 +2524,36 @@ checksum = "2a60c7ce501c71e03a9c9c0d35b861413ae925bd979cc7a4e30d060069aaac8d"
...
  "unicase 2.6.0",
 ]
 
 [[package]]
 name = "minidom"
-version = "0.13.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "332592c2149fc7dd40a64fc9ef6f0d65607284b474cef9817d1fc8c7e7b3608e"
+version = "0.14.0"
+source = "git+https://github.com/lrubiorod/xmpp-rs-fork#c0b6a6eaf7b03fa043c5b03e5aaabef897e61ddf"
```
Comment (aesedepece) [inline Cargo.lock:2538]: Forked repos should belong to Witnet organization to reduce obscurity.

**Review by aesedepece — CHANGES_REQUESTED** [review]

File: Cargo.lock (hunk 1 above)
Comment (tmpolaczyk) [reply Cargo.lock:2538]: Let's move it to https://github.com/witnet/xmpp-rs-fork

File: Cargo.lock [hunk 2]
```diff
@@ -9,54 +9,54 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "3720d0064a0ce5c0de7bd93bdb0a6caebab2a9b5668746145d7b3b0c5da02914"
 dependencies = [
  "actix-rt",
- "bitflags 1.2.1",
```
Comment (tmpolaczyk) [inline Cargo.lock:12]: This Cargo.lock has too many changes, I think you accidentally updated some other dependencies? You can revert it using

    git checkout upstream/master -- Cargo.lock

And compiling again.

**Comment by aesedepece** [comment]
@lrubiorod the last commit is really interesting :ok_hand:

**Comment by aesedepece** [comment]
@lrubiorod the last commit is super interesting! Can you provide an example of how the

**Comment by aesedepece** [comment]
Sorry, I accidentally closed this. Reopening!

**Comment by lrubiorod** [comment]
> @lrubiorod the last commit is really interesting ok_hand will you be explicit about this on the WIP?

Yes, I have to include it

**Review by aesedepece — APPROVED** [review]

**Pull request merged**